admin-dev/themes/new-theme/js/pages/product/product-map.js

```diff
@@ -116,5 +116,19 @@ export default {
     configuration: {
       fileManager: '.openfilemanager',
     },
+    photoswipe: {
+      element: '.pswp',
+    },
+    dzTemplate: '.dz-template',
+    dzPreview: '.dz-preview',
+    sortableContainer: '#product-images-dropzone',
+    sortableItems: 'div.dz-preview:not(.disabled)',
+    dropzoneContainer: '.dropzone-container',
+    checkbox: '.md-checkbox input',
+    shownTooltips: '.tooltip.show',
+    savedImage: (imageId) => `.dz-preview[data-id="${imageId}"]`,
```

Comment:
```suggestion
    dropzonePreview: (imageId) => `.dz-preview[data-id="${imageId}"]`,
```

Reply:
I don't like this one, it's more precise than juste the dropzonePreview, also it's already inside the dropzone attribute so you would call it `preview`, but it's precisely an certain image, do you have another suggestion?